<?xml version="1.0"?>
<launch>
   <!-- RGBD Server Launcher -->
  
   
   <!-- set debug mode, camera quality and baxter face control -->
   <arg name="quality" default="sd" />  <!--only with kinect2 sd/qhd/hd -->
   <arg name="debug" default="true" />
   <arg name="camera_topic_debug" default="/kinect2/$(arg quality)/points" /> <!-- kinect_head/kinect_chest/kinect_extern/openni2/kinect2 topics -->
   <arg name="baxter_face_ctrl" default="true" />
   <arg name="publish_output_pc" default="true" />
   
    <!-- set params for the servers -->
   <param name="quality" value="$(arg quality)" />  <!--only with kinect2 sd/qhd/hd -->
   <param name="debug" value="$(arg debug)" />
   <param name="baxter_face_ctrl" value="$(arg baxter_face_ctrl)" />
   <param name="camera_topic_debug" value="$(arg camera_topic_debug)" /> <!-- kinect_head/kinect_chest/kinect_extern/openni2/kinect2 topics -->
   <param name="publish_output_pc" value="$(arg publish_output_pc)" />
  

  <!-- Running the RGBD Server -->
  <node pkg="apc_objects_detection_rgbd" type="rgbd_server" name="lookforobject_rgbd" args="-t" output="screen" respawn="true"/>
  
</launch>
